application/services/event_service.py

```
@@ -1,30 +1,22 @@
-            response.raise_for_status()
-            all_events = response.json()
-            return all_events.get("data", {}).get("events", [])
-        except (RequestException, ValueError) as e:
-            logger.error(
-                f"Error fetching CMS content item for {self.get_cms_endpoint()}: {e}"
-            )
+            with open(self.data_file_path, "r") as file:
+                all_events = yaml.safe_load(file)
+                return all_events.get("events", [])
+        except (FileNotFoundError, yaml.YAMLError) as e:
+            logger.error(f"Error fetching content item for {self.data_file_path}: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)